pkg/ccl/backupccl/split_and_scatter_processor.go

```diff
@@ -82,22 +92,26 @@ func (s dbSplitAndScatterer) split(
 	if s.kr == nil {
 		return errors.AssertionFailedf("KeyRewriter was not set when expected to be")
 	}
-	if s.db == nil {
+	if s.sas == nil {
 		return errors.AssertionFailedf("split and scatterer's database was not set when expected")
 	}
 
-	expirationTime := s.db.Clock().Now().Add(time.Hour.Nanoseconds(), 0)
+	expirationTime := s.sas.Clock().Now().Add(time.Hour.Nanoseconds(), 0)
 	newSplitKey, err := rewriteBackupSpanKey(codec, s.kr, splitKey)
 	if err != nil {
 		return err
 	}
 	if splitAt, err := keys.EnsureSafeSplitKey(newSplitKey); err != nil {
-		// Ignore the error, not all keys are table keys.
+		// The key might be corrupt, and therefore we cannot guarantee it is a valid
+		// split key. The restore can still continue without this split. This error
+		// is not expected after #109483 is fixed.
+		log.Errorf(ctx, "failed splitting at key %s err: %v", newSplitKey, err)
```

Comment:
Minor, but given your changes to EnsureSafeSplitKey, this will be logging the key twice since err also contains it.

Reply:
done, thank you.